src/averageAges.js

```diff
@@ -15,11 +15,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const male = arguments.length < 2
+    ? filteredByGender(people, 'm')
+    : filteredByGender(people, 'm').filter(
+      obj => Math.ceil(obj.died / 100) === century);
```

Comment:
I guess, it would be better to move `filteredByGender(people, 'm')` out of ternary operator, not to repeat the same. Like this:
```suggestion
  const male = arguments.length < 2
    ? people
    : people.filter(obj => Math.ceil(obj.died / 100) === century);
      
      men = filteredByGender(people, 'm');
```
But it's up to you))